main.sh

```diff
@@ -69,7 +69,9 @@ fi
 
 /usr/games/cowsay "Running with the flag $phpcs_standard"
 
+[[ ":1:yes:true:" = *:$SKIP_LINT:* ]] && lint_option='--lint=false' || lint_option='--lint=true'
```

Comment:
It would be better to keep the check binary: `true/false`, in line with vip-go-ci flag input.
Additionally, if you want, you can convert the input value of the flag to lowercase before running the check, so that even if a user has set the value in upper, lower or mixed case, the check passes according to the value.